# Define the goal
geometry_msgs/PoseStamped global_goal  
---
# Define the result
bool finished 
std_msgs/UInt8 replan_number
std_msgs/UInt8 emergency_stop_times # Maybe we can put a custom message with the list of the positions, duration etc.
std_msgs/Duration time_spent
bool not_possible

---
# Define a feedback message
std_msgs/Float32 distance_to_goal
std_msgs/UInt8 global_waypoint 
std_msgs/Float64 travel_time
std_msgs/String percent achieve
std_msgs/String ETA
